app/src/test/java/org/glucosio/android/adapter/BackupAdapterTest.java

```diff
@@ -47,6 +47,9 @@ public void ReturnViewWithSizeFormatted_WhenAsked() throws Exception {
         View view = adapter.getView(0, null, viewGroup);
 
         TextView sizeView = (TextView) view.findViewById(R.id.item_history_type);
-        assertThat(sizeView).hasText("1.00KB");
+        String sizeViewText = sizeView.getText().toString();
+        assertTrue(sizeViewText.matches(
+                "([0-9]*\\.[0-9]+[0-9]+[K]+[B])|([0-9],[0-9]+[0-9]+[K]+[B])"
```

Comment:
Well, this a bit tricky regexp is used because in some countries, at least mine (Poland) we use comma in number expression  instead of dot. That was a reason why unit tests failed locally on my computer.

To avoid problem with `Locale` I decided to use regExp which checking both `1.00KB` and `1,00KB`

Reply:
I would just force locale for test

Reply:
I don't like idea of forcing anything just for test purpose. Moreover, I really don't like hardcoded values for the same purpose.

But ok, I can change it from regExp checking format of given value to forcing Locale just to get a specific value described in test scenario.

Reply:
Let's discuss in the chat. I asked to force locale in the test for test. So the production code stays untouched.

There we could also discuss what do you mean by `hardcoded values`